src/app/components/drawer/Projects/ProjectsComponent.js

```diff
@@ -61,7 +60,6 @@ const ProjectsComponent = ({
       setActiveItem({ type: path[2], id: parseInt(path[3], 10) });
     }
   }, [location.pathname]);
-  const isActive = (type, id) => type === activeItem.type && id === activeItem.id;
```

Comment:
I think this was the line creating the problem... it creates derived-state that was triggering a second render.